lib/tasks/index_timestamp.rake

```diff
@@ -0,0 +1,7 @@
+namespace :index_timestamp do
+  desc "Check the timestamp for the last index update and raise an error if it has not updated today"
+  task check: :environment do
+    active_manager = IndexManager.all.first { |index_manager| !index_manager.solr_collection.match(/rebuild/) }
```

Comment:
Can we check for solr-collection `catalog-alma-production`? 
Sometimes we might run an index test with a different solr-collection. This will create an index manager instance and because it won't have the '-rebuild' in the name, it will alert us for the wrong index manager instance.

Reply:
Done.